Makefile.toml

```diff
@@ -89,8 +93,13 @@ toolchain = "nightly"
 command = "cargo"
 args = ["udeps", "--all-targets"]
 
+[tasks.machete]
```

Comment:
Curiosity: why both cargo-machete and cargo-udeps ?

Reply:
cargo machete also inspect the other crates and I saw that both of them didn't recognize some unused crates ... to summarize
- `udeps` didn't catch `digest`
- `machete` didn't catch `sha2`
- rustrover catch both but also a false positve `hex-literal`

Reply:
Anyway I've remove it on remote-ci. Just enable locally otherwise the CI become too slow.